lib/client.js

```diff
@@ -379,12 +379,18 @@ class AssetComputeClient extends EventEmitter {
         return response;
     }
 
-    async checkEventJournal(journalUrl) {
+    async isEventJournalReady(journalUrl) {
         const ioEvents = new AdobeIOEvents({
             accessToken: this.accessToken,
             orgId: this.integration.technicalAccount.org
         });
-        await ioEvents.getEventsFromJournal(journalUrl);
+        try {
+            await ioEvents.getEventsFromJournal(journalUrl);
+            return true;
+        } catch(e) {
+            console.log(e);
```

Comment:
Not sure if we want to log anything here if this is an expected code path and returns false while not ready. Otherwise it looks like lots of errors are happening.

Do we know the type of error that is expected while it's not ready? Maybe check for that one and only log if it's an (unexpected) different error.

Reply:
I assume it is 500 internal server error if it is not fully ready (or at least that's what I encountered in practice). And probably 404 if there is no journal created yet...(we should have a unit test for this)

also I agree, we should not log the error since it is expected sometimes

Reply:
+1. We had proof during the hackathon that logging properly handled errors is confusing for developers/people not as familiar with the code.

Reply:
@alexkli  currently IO Events throws 500, but I think they plan to change it in future. So I won't prefer using the error type yet. Okay to not log error. I remember eslint complaining `'e' is defined but never used.eslintno-unused-vars`

Reply:
@pheenomenon You can disable the eslint check for that line: `// eslint-disable-line no-unused-vars` on the same line as the `catch` statement.